public/eligibleUsers.json

```diff
@@ -0,0 +1,60 @@
+[
```

Comment:
As discussed, this file should be managed either in Sanity or in the database to avoid having to deploy the project at each update.

Reply:
next PR